themes/media-theme-netflix.js

```diff
@@ -0,0 +1,164 @@
+import MediaTheme from './media-theme.js';
```

Comment:
Fine for now, but there are several inconsistencies with the current browser-based netflix player. Should make followup issue(s) to clean up. Also may want to consider splitting out the part that sizes this to full width from the rest of the theme.